lambdas/pkgpush/src/t4_lambda_pkgpush/__init__.py

```diff
@@ -606,7 +609,8 @@ def create_package(req_file):
                     )
                 )
             else:
-                pkg.set(logical_key, str(physical_key), meta)
+                pkg.set(logical_key, str(physical_key))
+                pkg[logical_key]._meta = meta or {}
```

Comment:
I know that smells but I don't think I could do better without major refactoring.

Reply:
ok 🤷